backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1758,3 +1758,75 @@ databaseChangeLog:
             columns:
               - column:
                   name: test_result_delivery
+  - changeSet:
+      id: add-spring-session-jdbc-support
+      author: emma@skylight.digital
+      comment: Add a table to back spring sessions.
+      changes: 
+        - createTable:
+              tableName: spring_session
+              remarks: Data store for user sessions, as created by SpringSession.
+              columns:
+                - column:
+                    name: primary_id
+                    type: *idtype
```

Comment:
Canonically this is a `char(36)`, not a UUID—is Spring Session actually using a UUID, and they just haven't updated their example DDL to reflect postgresql having a UUID type? 🤣

Reply:
Sadly not! At least not from [this example](https://github.com/spring-projects/spring-session/blob/main/spring-session-jdbc/src/main/resources/org/springframework/session/jdbc/schema-postgresql.sql). I updated to a `char(36)` instead 😃